src/main/kotlin/no/nav/klage/oppgave/api/mapper/KlagebehandlingMapper.kt

```diff
@@ -21,12 +20,15 @@ class KlagebehandlingMapper(
     }
 
     fun mapKlagebehandlingToKlagebehandlingView(klagebehandling: Klagebehandling): KlagebehandlingView {
-        
+
+
+        //TODO: Trenger vi egentlig Mottak her også?
+        //Skal vi vise frem dataene vi mottok fra førsteinstans i tillegg til det saksbehandler har satt/endret?
         return KlagebehandlingView(
             id = klagebehandling.id,
             klageInnsendtdato = klagebehandling.innsendt,
             //TODO get name from norg2
-            fraNAVEnhet = klagebehandling.mottak.avsenderEnhet ?: "mangler",
+            fraNAVEnhet = klagebehandling.avsenderEnhetFoersteinstans,
```

Comment:
Mulig at avsenderEnhetFoersteinstans ikke skulle vært dratt over i klagebehandling..?

Reply:
Spørsmålet er vel hva mottak skal være. Er det en ren historikktabell, eller har vi en referanse tilbake som vi skal bruke? For vi har bruk for avsender-enheten senere, og det er kanskje ryddigere da å ha den på klagebehandlingen enn å referere tilbake til mottaket.

Reply:
Kan være litt enig der. Kanskje passer bedre med den infoen i "Mottak".

Reply:
Sååå, der sa dere vel det motsatte av hverandre? Godt å høre at usikkerhet ikke bare ligger inni mitt hode da, for jeg vet ikke sikker på hva som var riktig her.. :)

Reply:
Hehe, når jeg tenker meg om så er det vel kanskje like greit det Andreas sier. For å finne avsender sjekker vi mottaket. Tror det er greit at den ligger der.

Reply:
Vi har jo en-til-en mellom klagebehandling og mottak, og jeg tenkte bare at det ga mening å ha "avsender" i et "mottak".

Reply:
Okei, da endrer jeg det til å være sånn.